index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="page">
+    <header class="menu">
+      <nav>
+        <img src="images/menu.svg" alt="Menu">
+      </nav>
+    </header>
+    <section>
+        <div class="headline">
+          <span class="background-image">
+            <!-- <img src="images/header.svg" alt=""> -->
+          </span>
+          <h1 class="greeting"> Hey there. <br>
+            I'm Philemon <br>
+            <span class="job">I'm a Software Developer</span>
+          </h1>
+            <p class="description"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hesitate to contact me.</p>
+            <ul class="media">
```

Comment:
- Kindly use the appropriate color for the text it will make your design stick to the Figma design :+1: 

## Your design 

![image](https://github.com/KiengPhyl/Philemon-portfolio/assets/95297251/f3281850-22b6-470c-a5db-862ea6d8266a)



## Figma design 

![image](https://github.com/KiengPhyl/Philemon-portfolio/assets/95297251/17a3b2f3-32ea-4846-a3af-e147c4f17beb)